hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/tools/federation/RouterAdmin.java

```diff
@@ -97,6 +106,7 @@
 public class RouterAdmin extends Configured implements Tool {
 
   private static final Logger LOG = LoggerFactory.getLogger(RouterAdmin.class);
+  private static final String DUMP_COMMAND = "-dumpState";
```

Comment:
I considered it, but it will make this patch much larger and riskier. Additionally, it really should be refactored to use the HDFS AdminHelper utility class. I've filed [HDFS-16856](https://issues.apache.org/jira/browse/HDFS-16856) to address the relevant refactoring.